designer/main.js

```diff
@@ -24,7 +24,8 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 var jsonParser = bodyParser.json();
 
-const adapter = new JSONFile('public/data/meta/db.json');
+//const adapter = new JSONFile('public/data/meta/db.json');
```

Comment:
This will have some problem. We need to keep two subfolders into the same mounted path. (public/data/meta for the storage of internal metadata, public/data/photo for the storage of uploaded images).